components/Header.styles.js

```diff
@@ -0,0 +1,26 @@
+import { StyleSheet } from 'react-native';
+import { COLORS } from '../theme/colors';
+import { SIZES } from '../theme/fonts';
+import { SPACINGS } from '../theme/spacings';
+
+export const styles = StyleSheet.create({
+  container: {
+    backgroundColor: COLORS.paper.main,
```

Comment:
on web this doesn't seem to get inherited by actionsContainer so the background is grey, on android it works. Do we need to care?

Reply:
nope :)

Reply:
or maybe you can check if it works on iOS as well?

Reply:
will do a check with the iphone but webview is often pretty far out

Reply:
hmm currently getting no projects loading on any branch so will merge and test on dev